ajax/libs/angular-tree-control/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "angular-tree-control",
+  "filename": "angular-tree-control.min.js",
+  "version": "0.2.26",
+  "description": "Angular Tree Control",
+  "homepage": "https://wix.github.io/angular-tree-control",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/wix/angular-tree-control.git"
+  },
+  "keywords": [
+    "angular-tree-control",
+    "Angular JS Tree",
+    "tree control component"
+  ],
+  "author": "Yoav Abrahami",
+  "license": "MIT",
+  "npmName": "angular-tree-control",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "angular-tree-control*",
+        "+(css|images)/!(sample).+(css|png|map)"
```

Comment:
I don't think combine css and images in the same rule is a good idea, they have difference naming pattern.